### PR TITLE
🆕 feat(I18n): Add I18n component

### PR DESCRIPTION
**Pull request by capdiem**

### CODE REVIEW

File: src/Component/BlazorComponent/Extensions/StringExtensions.cs [hunk 1]
```diff
@@ -15,8 +15,6 @@ public static string ReplaceFirst(this string str, string oldValue, string newVa
             return str;
         }
 
-        // todo: regex.replace is more fast?
```
Comment (doddgu) [inline src/Component/BlazorComponent/Extensions/StringExtensions.cs]: fix warnings，you can see them in review

**Review by doddgu — CHANGES_REQUESTED** [review]

File: src/Component/BlazorComponent/Extensions/StringExtensions.cs (hunk 1 above)
Comment (capdiem) [reply src/Component/BlazorComponent/Extensions/StringExtensions.cs]: fix all nullable warnings? Impossible...

Comment (doddgu) [reply src/Component/BlazorComponent/Extensions/StringExtensions.cs]: The review page shows 10 only. Fix as much as you see.

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**